qemu/qemu.go

```diff
@@ -2568,6 +2615,14 @@ func (config *Config) appendLogFile() {
 	}
 }
 
+func (config *Config) appendFwCfg() {
```

Comment:
please change the signature of this function to receive a `QMPLog`

Reply:
signature changed.